recipes/pygridtools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+    name: pygridtools
+    version: 0.2
+
+source:
+    git_url: https://github.com/Geosyntec/pygridtools.git
+    git_tag: v0.2
```

Comment:
I'll need to mull that over a bit. Uploading to pip always feels like a hassle to me. But that's mostly b/c i don't do it enough.